module-services/service-evtmgr/service-evtmgr/AppSettingsNotify.hpp

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
```

Comment:
Why is the file name AppSettingsNotify if it only notifies bluetooth audio settings?

Reply:
I was checking direct inter-services and applications dependencies. In this case, we have a direct call from service to application (which might not exist at the moment) - now if you disable Bluetooth-service code this is the direct dependency to it. This is why I moved it to a separate file.
By the way - services shouldn't know about applications and such actions should be rather made via sys::action - not via direct call.

I can remove it, it does literally nothing except moving a piece of code to a separate function (which enables us to turn it off in compile time)